frontend/src/components/waitlist/LocalePicker.tsx

```diff
@@ -47,9 +47,21 @@ async function importLanguages(locale: string): Promise<LocaleDisplayNames> {
     );
     return languages.main[locale].localeDisplayNames.languages;
   } catch (_e) {
-    const languagesEn = await import(
-      "cldr-localenames-modern/main/en/languages.json"
-    );
-    return languagesEn.main.en.localeDisplayNames.languages;
+    try {
+      // cldr-localenames-modern doesn't include suffixed locale codes for
```

Comment:
praise: Great job adding comments where it gets a bit complicated to follow. Truly helps a lot!